modules/checks/pingdom.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const path = require('path')
+const checkFunction = require('../check-function')
+
+module.exports = async () => {
+  if (no_cli_flags || options_keys.includes('--pingdom')) {
+    const name = 'Pingdom'
+    async function tryBlock(page) {
+      await page._client.send('Emulation.clearDeviceMetricsOverride')
+      await page.goto('https://tools.pingdom.com/')
+      await page.waitForSelector('#urlInput')
+      await page.type("#urlInput", url)
+      await page.click('.ng-tns-c3-0 > .view > .grid > .grid__col-sm-2 > .button')
+      await page.waitForSelector('div > .grid > .grid__col-md-8 > .button-container > .download-har')
```

Comment:
Is there only one button with `.download-har`? Because `>` selectors can easily break if there are changes in the markup in one of those selector parts.

Reply:
Good point. I simply copied this selector from devtools, but now that you mentioned it, its better to select just the class.